packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -55,19 +55,49 @@ import { getIllegalCharacters } from './getIllegalCharacter'
 const ROOT_DIR_ID = 'io.cozy.files.root-dir'
 const CONTENT_TYPE_OCTET_STREAM = 'application/octet-stream'
 
+/**
+ * Normalize a file, adding document's doctype if needed
+ *
+ * @param  {FileDocument} file - File to normalize
+ * @param  {FileAttributes} file.attributes - Attributes of file to keep
```

Comment:
I think that FileDocument already contains FileAttributes. No need to add it

Reply:
Je crois que `FileDocument` est défini via `@typedef` juste au dessus
Si on reste ainsi il serait en effet mieux de retrouver `FileAttributes` en `@property` dans la définition au dessus :)